client/src/modules/events/pages/eventsPage.tsx

```diff
@@ -75,7 +81,14 @@ export const EventsPage: NextPage = () => {
   return (
     <VStack>
       <Stack w={['90%', '90%', '60%']} maxW="600px" spacing={6} mt={10} mb={5}>
-        <Heading>Events: </Heading>
+        <Flex justifyContent={'space-between'} alignItems={'center'}>
+          <Heading>Events: </Heading>
+          {canAuthenticateWithGoogle && (
```

Comment:
Using the `GoogleAuthenticate` permission will show the buttons only to instance owners, what is probably not the intent. `useAuth()` can return now `isLoggedIn` property which would be more appropriate in these places. Or some new instance permission for accessing dashboard.

This applies to other pages too.

After updating branch with the changes from `main` couple things will need to be changed, as `useCheckPermission` is no longer there.